config/config.go

```diff
@@ -216,12 +347,15 @@ func getDefaultConfig() *Config {
 	config.Behavior.AutoFetch = true
 	config.Behavior.StashBeforeSwitch = true
 
-	// Default aliases
+	// Default simple aliases
 	config.Aliases["st"] = "status"
-	config.Aliases["co"] = "checkout"
 	config.Aliases["br"] = "branch"
 	config.Aliases["ci"] = "commit"
 
+	config.Aliases["ac"] = []interface{}{"add .", "commit tmp"}
```

Comment:
I noticed a bug while reviewing this implementation, so I created an issue. 🚀 
https://github.com/bmf-san/ggc/issues/120